frontend/src/helpers/destroyStore.ts

```
@@ -0,0 +1,12 @@
+import { getActivePinia, type Pinia, type Store, type StoreDefinition } from 'pinia';
+
+export function destroyStore(store: Store, pinia: Pinia = getActivePinia()!): void {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)